dashboard/agent.py

```diff
@@ -509,6 +510,21 @@ async def _check_parent():
         )
 
         loop = asyncio.get_event_loop()
+
+        def sigterm_handler():
+            logger.warning("Exiting with SIGTERM immediately...")
+            # Exit code 0 will be considered as an expected shutdown
+            os._exit(signal.SIGTERM)
```

Comment:
hmm does this return exit code 0?

Reply:
no. It has the same return code as dashboard (I think 15)